<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="authenticate_tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Вход</string>
    </attribute>
    <widget class="QLineEdit" name="loginUserID_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Введите ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginPassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Введите пароль</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Войти</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Регистрация</string>
    </attribute>
    <widget class="QPushButton" name="registrate_pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registratePassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Придумайте пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registrateUsername_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Придумайте имя</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registrateEmail_lineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Введите Вашу почту</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit2_pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
